### PR TITLE
Add implementation for CiviCRM Activity entities

### PR DESCRIPTION
**Pull request by dontub**
Frontend part for https://github.com/systopia/de.systopia.remoteactivity.

### CODE REVIEW

File: modules/civiremote_activity/civiremote_activity.routing.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+civiremote_activity.create_form:
+  path: '/civiremote/activity/{profile}/add'
```
Comment (jensschuppe) [inline modules/civiremote_activity/civiremote_activity.routing.yml]: I'd suggest `/civiremote/activity/add/{profile}` for following Drupal path logic more closely. Read: Identify the entity type (`civiremote/activity`), add the action to perform (`add`), add parameters needed for this action (`{profile}`)

File: modules/civiremote_activity/civiremote_activity.routing.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  options:
+    no_cache: TRUE
+    parameters:
+      profile:
+        type: string
+  requirements:
+    _user_is_logged_in: 'TRUE'
+
+civiremote_activity.update_form:
+  path: '/civiremote/activity/{profile}/{id}'
```
Comment (jensschuppe) [inline modules/civiremote_activity/civiremote_activity.routing.yml]: I'd suggest `/civiremote/activity/{id}/{profile}` or even `/civiremote/activity/{id}/update/{profile}` for following Drupal path logic more closely. Read: Identitfy the entity (`civiremote/activity/{id}`), add the action to perform (`update`), add parameters needed for this action (`{profile}`).

File: modules/civiremote_activity/src/Controller/ActivityCreateController.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+use Drupal\Core\Controller\ControllerBase;
+
+final class ActivityCreateController extends ControllerBase {
+
+  /**
+   * @phpstan-return array<int|string, mixed> JSON serializable.
+   */
+  public function form(string $profile): array {
+    $form = $this->formBuilder()->getForm(ActivityCreateForm::class);
+    $form['#title'] ??= $this->t('CiviRemote Activity Create');
```
Comment (jensschuppe) [inline modules/civiremote_activity/src/Controller/ActivityCreateController.php]: This should read *Add CiviCRM Activity*, following Drupal conventions of naming actions, and hiding the *CiviRemote* layer, which I think is irrelevant to the user.

File: modules/civiremote_activity/src/Controller/ActivityUpdateController.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+use Drupal\Core\Controller\ControllerBase;
+
+final class ActivityUpdateController extends ControllerBase {
+
+  /**
+   * @phpstan-return array<int|string, mixed> JSON serializable.
+   */
+  public function form(string $profile, int $id): array {
+    $form = $this->formBuilder()->getForm(ActivityUpdateForm::class);
+    $form['#title'] ??= $this->t('CiviRemote Activity Update');
```
Comment (jensschuppe) [inline modules/civiremote_activity/src/Controller/ActivityUpdateController.php]: This should read *Update CiviCRM Activity*, following Drupal conventions of naming actions, and hiding the *CiviRemote* layer, which I think is irrelevant to the user.

**Review by jensschuppe — CHANGES_REQUESTED** [review]

File: modules/civiremote_activity/civiremote_activity.routing.yml (hunk 1 above)
Comment (dontub) [reply modules/civiremote_activity/civiremote_activity.routing.yml]: Changed.

File: modules/civiremote_activity/civiremote_activity.routing.yml (hunk 2 above)
Comment (dontub) [reply modules/civiremote_activity/civiremote_activity.routing.yml]: Changed.

File: modules/civiremote_activity/src/Controller/ActivityUpdateController.php (hunk 4 above)
Comment (dontub) [reply modules/civiremote_activity/src/Controller/ActivityUpdateController.php]: Changed.

File: modules/civiremote_activity/src/Controller/ActivityCreateController.php (hunk 3 above)
Comment (dontub) [reply modules/civiremote_activity/src/Controller/ActivityCreateController.php]: Changed.

**Review by jensschuppe — APPROVED** [review]

**Pull request merged**